libs/design/src/molecules/select/option/option.directive.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  Directive,
+  HostBinding,
+  Optional,
+  TemplateRef,
+} from '@angular/core';
+
+/**
+ * A directive for styling an select item's content.
+ */
+@Directive({
+  selector: '[daffSelectOption]',
+})
+export class DaffSelectOptionDirective {
+  @HostBinding('class.daff-select__option') class = true;
```

Comment:
How does this work with the templating in the select component? This host binding isn't actually adding the class to `[daffSelectOption]`